src/ILLink.Tasks/LinkTask.cs

```diff
@@ -191,6 +191,13 @@ public class ILLink : ToolTask
 		public bool RemoveSymbols { set => _removeSymbols = value; }
 		bool? _removeSymbols;
 
+		/// <summary>
+		/// The version of the link task. Used to determine the meaning of certain properties.
+		/// Roughly maps to the TargetFrameworkVersion.
+		/// </summary>
+		[Required]
+		public int LinkVersion { get; set; }
```

Comment:
Works - but I would prefer if we simply passed something like TargetFrameworkVersion - could be the same numerical value, but make it clear that it's the TFM version - I don't want to introduce yet another version number to think about.

Reply:
I tried to make that work, but that means we have to do all of the mapping logic in MSBuild properties. That seems much harder to get right, and also very hard to test